docs/HISTORY.txt

```diff
@@ -5,7 +5,7 @@ Changelog
 2.1.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Show the folder_contents icon appear on 1.x and 2.x versions of ftw.simplelayout [Nachtalb]
```

Comment:
I don't understand this sentence. Maybe because of the two verbs "show" and "appear" in the sentence?

What about something like the following?

> Add missing icon for the folder contents action of simplelayout blocks.

Reply:
by brain did a 2 sentence race condition here 🤪

Reply:
I changed it

Reply:
🤣